### ISSUE TITLE
Fork?

### ISSUE THREAD

**Issue by Bebowi** [plietar/librespot#263]
Is michaelherger's fork the official one?

**Comment by sashahilton00** [comment]
I think the plan was for Paul to nominate someone else as a maintainer so that we don't lose al issues/PR's by moving to a fork

**Comment by Bebowi** [comment]
I don't know but when I took a look at Fork it seems like this master has been merged to michaelherger's

**Comment by michaelherger** [comment]
NO! I only forked it as the base to apply the few changes I need for my Spotty plugin. But I have close to zero Rust experience...

**Comment by Bebowi** [comment]
Anybody here who feels experienced enough and has time to do an official fork?

**Comment by dtcooper** [comment]
Any clear winners here?

**Comment by Bebowi** [comment]
At the moment there is nobody who wants to win. I'm new to Rust and I think many others too

**Comment by janLo** [comment]
How about handing it into an organization of its own and invite known contributors as members. That would enable others than @plietar to approve and merge PullRequests.

**Comment by dtcooper** [comment]
That's where I'm at too, re: just learning rust. Not a bad idea, @janLo. Is @plietar responding to any librespot noise at able to do that?

**Comment by kingosticks** [comment]
Nobody has stepped forward so there's nothing to respond to. I think that's the first step here.

**Comment by sashahilton00** [comment]
@Bebowi @michaelherger @janLo @dtcooper we've forked librespot into an organisation as a successor and have begun cleanup/implementing PR's and bug fixes. It's available at https://github.com/librespot-org/librespot
